Sources/BraintreeCoreSwift/BTAPIClient.swift

```diff
@@ -264,7 +285,7 @@ import Foundation
 
     /// :nodoc:
     @objc(POST:parameters:httpType:completion:)
-    public func post(_ path: String, parameters: [String: Any]? = nil, httpType: BTAPIClientHTTPTypeSwift, completion: @escaping RequestCompletion) {
+    public func post(_ path: String, parameters: [String: Any]? = nil, httpType: BTAPIClientHTTPService, completion: @escaping RequestCompletion) {
```

Comment:
Curious why the move to `String` here? What if our `POST` params include some bools or ints?

Reply:
All of our post dictionaries currently are [String: Any] types otherwise we would be getting errors during build since all Objective-C post requests are now using this Swift class. It seems like we could maybe cross that if we get to a point where a dictionary deviates from the currently established pattern but would likely be a larger discussion with gateway teams as this seems like the pattern based on all of their parameter dictionaries currently, unless I am missing a use case. Thoughts?

Reply:
Ah, I meant to comment this at the `[String: Any]` --> `[String: String]` change above in line `273` for our `GET`.

Reply:
Ah! I was working under the assumption that it was supposed to be unless our mock override is wrong in the implementation on main: https://github.com/braintree/braintree_ios/blob/master/UnitTests/BraintreeTestShared/MockAPIClient.swift#L26. I updated it to base it off of the mock implementation. If we think the mock has been wrong all along I can revert to `[String: Any]`